parsec/core/mountpoint/manager.py

```diff
@@ -300,15 +304,7 @@ async def mountpoint_manager_factory(
     if sys.platform == "win32":
         cleanup_parsec_drive_icons()
     elif sys.platform == "darwin":
-        try:
-            await cleanup_macos_mountpoint_folder(base_mountpoint_path)
-        except FileNotFoundError:
-            pass
-        except CalledProcessError:
-            # The unmount can fail if the app wasn't properly closed and will
-            # block login until reboot. Cleanup is skipped in such cases and
-            # should work properly after the next user's reboot.
-            pass
+        await cleanup_macos_mountpoint_folder(base_mountpoint_path)
```

Comment:
Comment about `CalledProcessError` exception has been removed, is it because it was incorrect or because the behavior is no longer true ?

Reply:
My mistake, I thought it was a duplicate of the comment inside the cleanup function. I'll add the missing information in the latter now that the exceptions have moved